bin/reth/src/stage/run.rs

```diff
@@ -192,21 +190,28 @@ impl Command {
                     )
                 }
                 StageEnum::TxLookup => (Box::new(TransactionLookupStage::new(batch_size)), None),
+                StageEnum::AccountHashing => {
+                    (Box::new(AccountHashingStage::new(1, batch_size)), None)
```

Comment:
because we want to erase the type here:
https://github.com/paradigmxyz/reth/blob/9881d6763465265dec99739bff04498844b37e80/bin/reth/src/stage/run.rs#L125